make/langtools/netbeans/nb-javac/build.xml

```diff
@@ -192,20 +208,7 @@
 
     <target name="publish-to-maven-central" 
             description="Build nb-javac and stage for maven central"
-            depends="clean,jar,zip-nb-javac-sources">
-
-        <zip basedir="../../../../" destfile="dist/nb-javac-${nb-javac-ver}-doc.zip">
-            <include name="README.md"/>
-            <include name="SECURITY.md"/>
-            <include name="LICENSE.txt"/>
-            <include name="CONTRIBUTING.md"/>
-        </zip>
```

Comment:
You need a javadoc Artifact to publish to maven central - it can be empty, but it must be present. This drops the required file.

Reply:
I wanted to avoid the duplicated code in the two targets. I moved in into `-prepare-maven` target.